<!DOCTYPE html>
<html lang="en"><head>
   <!-- Required meta tags -->
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
   <!-- Bootstrap CSS -->
   <link rel="stylesheet" href="./bootstrap.min.css">
   <!-- Font Awesome CSS -->
   <link rel="stylesheet" href="./all.min.css">
   <!-- Template CSS -->
   <link rel="stylesheet" href="./theme.min.css">
   <link rel="stylesheet" href="./custom.css">
   <title>Blank Page</title>
<script src="https://templates.lcs.brightspace.com/lib/assets/js/scripts.min.js" defer=""></script></head><body class="template-fallback"><div class="splash-container">
<div class="container-fluid"><br>
<div class="row">
<div class="col-12 offset-md-1 col-md-10 offset-lg-2 col-lg-8">
<h1 aria-hidden="true">DNA Organization in Eukaryotes and Prokaryotes</h1>
<div class="d2l-element mceNonEditable" data-idx="3" data-type="noicon" role="section" aria-describedby="d2l-uid-314">
<p id="d2l-uid-314" class="sr-only">Callout</p>
<div class="card card-standard" data-prop="0|noicon">
<div class="card-body">
<div class="card-text">
<h4 data-prop="1|null"><!-- Empty --></h4>
<div data-prop="2|null">
<p style="text-align: justify;"><strong><span style="font-size: 24px;">The Packing of Eukaryotic DNA<br></span></strong><br>Imagine that you are trying to store a garden hose. The hose takes up much less&nbsp;space if you coil it neatly around a spool. This is exactly what happens in eukaryotic&nbsp;cells. DNA is wound around special proteins, called histones, that act like spools&nbsp;Histones are positively charged proteins, and the negatively charged DNA strands&nbsp;are attracted to them. This attraction causes the DNA strands to wrap tightly around&nbsp;a cluster of eight histones, greatly reducing the amount of space that the DNA occupies. Each unit, consisting of eight histone proteins and the wrapped DNA, is called&nbsp;a nucleosome (Figure 1). Stretches of DNA, called linkers, connect one nucleosome to&nbsp;the next. At this level of organization, the strands of chromatin appear like a string&nbsp;of beads.&nbsp;The volume of the DNA has now been reduced by a factor of seven. Further&nbsp;packing of the DNA is accomplished by coiling strings of nucleosomes into cylindrical fibers with a diameter of 30 nm (3.0 × 10–10 m). These are commonly known&nbsp;as 30 nm chromatin fibers, but they are also called solenoids. Each coil of a solenoid contains six nucleosomes<br>(Figure 1).&nbsp;Most of the DNA in the nucleus during the interphase stage is in the form of&nbsp;loosely packed nucleosomes or solenoids. If the DNA contains active genes, the&nbsp;nucleosomes must be altered to allow the proteins and enzymes that are needed for&nbsp;gene expression to access the coding sequences of the bases. Histone molecules may&nbsp;even assist in the regulation of this process. If the DNA contains inactive genes or&nbsp;non-coding sections of genes, the solenoids function as compact storage units that&nbsp;protect the DNA from potential damage. When a cell enters the reproductive stage&nbsp;of its life cycle, the solenoids can be further supercoiled to form the typical X-shaped&nbsp;chromosomes that are visible during the metaphase stage of mitosis.</p>
</div>
</div>
</div>
</div>
</div>
<hr>
<div class="d2l-element mceNonEditable" data-idx="3" data-type="noicon" role="section" aria-describedby="d2l-uid-480">
<p id="d2l-uid-480" class="sr-only">Callout</p>
<div class="card card-standard" data-prop="0|noicon">
<div class="card-body">
<div class="card-text">
<h4 data-prop="1|null"><!-- Empty --></h4>
<div data-prop="2|null">
<p style="text-align: justify;"><strong><span style="font-size: 24px;">Prokaryotic DNA Organization<br></span></strong><br>The organization of DNA in both eubacteria and archaea is simpler and has other&nbsp;features that set it apart from eukaryotic DNA. In this section, however, the discussion&nbsp;will be limited to DNA organization in eubacteria. The bacterial DNA consists almost&nbsp;entirely of one chromosome that is commonly circular (Figure 2). Linear chromosomes do occur in some forms of bacteria, but they are rare. Unbound by a nuclear&nbsp;membrane, the DNA is less tightly bound and more easily accessed by the enzymes and&nbsp;other molecules involved in replication than eukaryotic DNA&nbsp;In addition to the primary circular DNA, smaller circular pieces of DNA float&nbsp;throughout the cell. In a process called conjugation, these smaller circular pieces of&nbsp;DNA, called plasmids, are able to exit one cell and enter another. When two bacteria&nbsp;are close together, a plasmid in one bacterium can pass into the other. The recipient&nbsp;bacterium incorporates the new plasmid into its genome (Figure 3). This feature allows&nbsp;natural variation of bacteria and is very useful for genetic engineering research since it<br>allows scientists to insert plasmids with desired genes into host bacteria. (You will learn about&nbsp;genetic engineering in Chapter 8).&nbsp;<br>Despite the simplicity of their genomes, prokaryotes face the same problem as eukaryotes when packing their DNA into a small cell. Whereas eukaryotes have additional proteins that package their DNA, prokaryotes use a coiling technique. Imagine (or try) twisting an elastic band so that it forms tiny coils along its length. Then keep twisting the elastic band until the coils bunch on top of each other and form a tight ball. This formation happens with a bacterial genome and is called supercoiling. Whether DNA is packed using a complex arrangement (as in eukaryotes) or a supercoil (as in prokaryotes), nature has evolved different strategies to fit so much information into&nbsp;such a small space.<br>Telomeres: A Key Difference</p>
<p style="text-align: justify;">The structure of eukaryotic DNA presents another problem: DNA loss during replication. As you learned in Section 6.4, DNA polymerase produces small pieces of DNA&nbsp;(the Okazaki fragments) in the 59 to 39 direction of the lagging strand. The 59 end of&nbsp;each Okazaki fragment is initially attached to an RNA primer (Figure 4(a)). The last&nbsp;Okazaki fragment on the lagging strand begins at the last RNA primer position, which&nbsp;is located close to, but not at, the end of the strand. Normally, the DNA being assembled&nbsp;as part of each following Okazaki fragment meets up with an RNA primer. DNA poly-<br>erase attaches to the end of the DNA fragment and then removes and replaces the&nbsp;RNA primer. However, this does not happen to the primer on the final Okazaki fragment, since no DNA is adjacent to the 59 end. Instead, the RNA primer is removed but&nbsp;not replaced. Thus, no DNA is assembled on the lagging strand beyond the position of&nbsp;the last primer. Since the last sequence at the end of the parent strand is not copied, the&nbsp;final DNA is shorter by the amount of this uncopied segment.&nbsp;</p>
</div>
</div>
</div>
</div>
</div>
</div>
<div class="col-12"><footer><!-- <p>© [Client] [Year]</p> -->
<p><img src="./EVA%20SCHOOL1.png" alt="" title="" data-d2l-editor-default-img-style="true" style="max-width: 100%;"></p>
</footer></div>
</div>
</div>
</div>
<p>
<script src="./jquery-3.5.1.min.js"></script>
<script src="./popper.min.js"></script>
<script src="./bootstrap.min.js"></script>
</p>
<!-- Template JavaScript -->
<p>
<script src="./scripts.min.js"></script>
</p></body></html>