<!DOCTYPE html>
<html lang="en"><head>
   <!-- Required meta tags -->
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
   <!-- Bootstrap CSS -->
   <link rel="stylesheet" href="./bootstrap.min.css">
   <!-- Font Awesome CSS -->
   <link rel="stylesheet" href="./all.min.css">
   <!-- Template CSS -->
   <link rel="stylesheet" href="./theme.min.css">
   <link rel="stylesheet" href="./custom.css">
   <title>Blank Page</title>
<script src="https://templates.lcs.brightspace.com/lib/assets/js/scripts.min.js" defer=""></script></head><body class="template-fallback"><div class="splash-container">
<div class="container-fluid"><br>
<div class="row">
<div class="col-12 offset-md-1 col-md-10 offset-lg-2 col-lg-8">
<h1 aria-hidden="true">DNA Sequencing and Genetic Engineering: Redesigning Life Part 1</h1>
<p>&nbsp;</p>
<div class="d2l-element mceNonEditable" data-idx="3" data-type="noicon" role="section" aria-describedby="d2l-uid-243">
<p id="d2l-uid-243" class="sr-only">Callout</p>
<div class="card card-standard" data-prop="0|noicon">
<div class="card-body">
<div class="card-text">
<h4 data-prop="1|null"><!-- Empty --></h4>
<div data-prop="2|null">
<p style="text-align: center;"><span style="font-size: 24px;"><strong>Introduction</strong></span></p>
<p style="text-align: justify;">At one time, lake sturgeon was abundant in many lakes across Canada.</p>
<p style="text-align: justify;">Currently, however, due to overfishing for their meat and caviar, sturgeon populations are on the decline in many areas.&nbsp;</p>
<p style="text-align: justify;">There are serious penalties for fishing and eating this endangered species of fish, and conservation officers now have a reliable tool to help them enforce the ban: DNA.</p>
<p style="text-align: justify;">The lake sturgeon is a protected species under the federal Fisheries Act, and violators of hunting and fishing regulations can be prosecuted.&nbsp;</p>
<p style="text-align: justify;">If conservation officers come across campers eating fish, and they suspect that the fish is not in season or is an endangered species, they can take a small sample to a forensic lab.&nbsp;</p>
<p style="text-align: justify;">If the DNA matches that of a banned species, the conservation officers have sufficient evidence to give to the police.</p>
<p style="text-align: justify;">Trent University, in Ontario, has a Wildlife DNA Profiling and Forensics Laboratory.&nbsp;</p>
</div>
</div>
</div>
</div>
</div>
<div class="d2l-element mceNonEditable" data-idx="3" data-type="noicon" role="section" aria-describedby="d2l-uid-409">
<p id="d2l-uid-409" class="sr-only">Callout</p>
<div class="card card-standard" data-prop="0|noicon">
<div class="card-body">
<div class="card-text">
<h4 data-prop="1|null"><!-- Empty --></h4>
<div data-prop="2|null">
<p style="text-align: justify;">One of the major duties of this laboratory is to track and monitor endangered species and support the enforcement of the Convention on International Trade in Endangered Species of Wild Flora and Fauna (CITES).&nbsp;</p>
<p style="text-align: justify;">Thanks to a procedure called the polymerase chain reaction (PCR), developed in 1983 by Kary Mullis in California, conservation officers and forensic scientists do not need a large sample of DNA in order to run extensive DNA fingerprinting tests.&nbsp;</p>
</div>
</div>
</div>
</div>
</div>
<hr>
<p class="small"><span style="font-size: 24px;"><strong>The Polymerase Chain Reaction</strong> </span></p>
<div class="card card-standard">
<div class="card-body">
<div class="card-text">
<p style="text-align: justify;">PCR is DNA replication of a specific region of the genome, outside of the nucleus of a cell, in a laboratory setting.&nbsp;</p>
<p style="text-align: justify;">The whole process takes place in a microfuge tube (a small test tube) and consists of three steps: denaturation, annealing, and elongation.&nbsp;</p>
</div>
</div>
</div>
<div class="card card-standard">
<div class="card-body">
<div class="card-text">
<p style="text-align: justify;">These steps are repeated in many cycles, usually 30 to 40.</p>
<p style="text-align: justify;">In the first step, called denaturation, a double-stranded DNA molecule is denatured, or separated, into its two single strands.&nbsp;</p>
<p style="text-align: justify;">This occurs when the DNA strand is heated to 94 to 96 °C for 20 to 40 s, causing the hydrogen bonds holding the two&nbsp;strands together to break.&nbsp;</p>
<p style="text-align: justify;">The second step, called annealing, takes place at a lower temperature, from 50 to 65 °C, for 20 to 40 s.</p>
<p style="text-align: justify;">This step facilitates the annealing of synthesized single-stranded&nbsp;DNA primers to each of the separated single strands.&nbsp;</p>
<p style="text-align: justify;"><strong>DNA primer</strong> is a short single-stranded DNA sequence, easily synthesized in a laboratory, that is complementary to the sequence at one end of the target sequence.</p>
</div>
</div>
</div>
<div class="card card-standard">
<div class="card-body">
<div class="card-text">
<p style="text-align: justify;">The primers contain the nucleic acid sequence that is complementary to one end of the targeted fragment&nbsp;of DNA.&nbsp;</p>
<p style="text-align: justify;">Two DNA primers are used: one must be complementary to a short portion of DNA on one strand at, or near, a 3’ end of the target sequence, and the other primer must be complementary to a portion of DNA on the other strand at the opposite&nbsp;3’ end of the target strand.&nbsp;</p>
<p style="text-align: justify;">In the third step, called elongation or extension, two new DNA strands are synthesized at a temperature of 72 °C</p>
<p style="text-align: justify;">The two original DNA strands, which were separated in the first step, now act as templates.&nbsp;</p>
<p style="text-align: justify;">DNA polymerase binds to the primer-template hybrids and moves along each template, adding nucleotides that are complementary to the sequence until it reach the 59 end of the template.</p>
<p style="text-align: justify;">A very specific DNA polymerase, called Taq polymerase, is needed for PCR elongation.&nbsp;</p>
</div>
</div>
</div>
<hr>
<div class="stacked-panels row">
<div class="card col-12">
<div class="card-body">
<p></p>
<p style="text-align: justify;">Taq polymerase is extracted from the bacterium Thermus aquaticus.&nbsp;</p>
<p style="text-align: justify;">This bacterium is found in hot springs, so its enzymes can withstand high temperatures.&nbsp;</p>
<p style="text-align: justify;">The use of Taq polymerase is vital since it is not denatured by the high temperature that is used during the first step of PCR.&nbsp;</p>
<p style="text-align: justify;">This would not be the case with DNA polymerase&nbsp;from a typical organism.</p>
<p style="text-align: justify;">After the two double-stranded DNA sequences have been made, the process can run through another cycle.</p>
<p style="text-align: justify;">The temperature i elevated back to 94 °C, and the strands are denatured again. Four strands are used as templates&nbsp;and duplicated to produce new copies.&nbsp;</p>
<p style="text-align: justify;">This cycle is repeated 30 to 40 times, resulting in a high number of copies of the specific target sequence.</p>
<p style="text-align: justify;">By the third cycle, two of the eight DNA molecules exactly match the target molecules.</p>
</div>
</div>
<div class="card col-12">
<div class="card-body">
<p></p>
<p style="text-align: justify;">The small piece of DNA at each end of the target sequence is eventually lost, and the target DNA sequence vastly outnumbers any other sequence.&nbsp;</p>
<p style="text-align: justify;">Within 20 cycles, the PCR reaction delivers over one million copies of the DNA target sequence.&nbsp;</p>
<p style="text-align: justify;">&nbsp;Since each cycle, regulated simply by increasing&nbsp;and decreasing the temperature, takes only about 5 min, millions of copies of a DNA sequence can be synthesized in less than 2 hours.&nbsp;</p>
<p style="text-align: justify;">PCR is such an efficient process that it is used almost exclusively to amplify a target DNA sequence.&nbsp;</p>
<p style="text-align: justify;">It is the process that is used by Fish and Wildlife personnel to identify a species of fish from a minute sample.&nbsp;</p>
<p style="text-align: justify;">The equipment that performs the rapid temperature changes and the various enzymes and primers are all available commercially.</p>
</div>
</div>
</div>
<hr>
<div class="stacked-panels row">
<div class="card col-12">
<div class="card-body">
<p></p>
<p style="text-align: justify;">As you have learned, PCR can make millions of copies of a single segment of DNA very quickly. &nbsp;</p>
<p style="text-align: justify;">This means that only a very small amount of a biological sample is required.&nbsp;</p>
<p style="text-align: justify;">There is enough DNA in a hair follicle, a tiny blood spatter, or a tiny piece of fish tissue to perform PCR.&nbsp;</p>
<p style="text-align: justify;">Once the DNA of interest has been amplified using PCR, it is analyzed using different methods.</p>
</div>
</div>
<div class="card col-12">
<div class="card-body">
<p><strong></strong></p>
<p style="text-align: center;"><span style="font-size: 24px;"><strong>Gel Electrophoresis</strong> </span></p>
<p style="text-align: justify;">Once the targeted DNA sequence has been amplified by PCR, it needs to be separated and purified from the rest of the contents in the test tube.&nbsp;</p>
<p style="text-align: justify;">A chromatography-like process, called gel electrophoresis, uses the physical and chemical properties of DNA&nbsp;to separate the fragments.</p>
<p style="text-align: justify;">Gel electrophoresis separates nucleic acids and proteins by their rate of migration through a gel.</p>
<p style="text-align: justify;">Agarose gel, made from agar (a polysaccharide from seaweed), is normally used for nucleic acids.</p>
<p style="text-align: justify;">Negatively charged DNA fragments travel through the pores of the gel, away from a negative electrode at the starting end and toward a positive electrode at the destination end.&nbsp;</p>
<p style="text-align: justify;">DNA is negatively charged, due to the phosphate groups, and is therefore attracted to a positive electrode. Because of the negative charge that is carried by the phosphate groups, each nucleotide is ionized.</p>
</div>
</div>
</div>
<hr>
<div class="stacked-panels row">
<div class="card col-12">
<div class="card-body">
<p></p>
<p style="text-align: justify;">A method for separating large molecules, such as DNA,&nbsp;RNA, and proteins.</p>
<p style="text-align: justify;">Once the targeted DNA sequence has been amplified by PCR, it needs to be separated and purified from the rest of the contents in the test tube.&nbsp;</p>
<p style="text-align: justify;">A chromatography-like process, called gel electrophoresis, uses the physical and chemical properties of DNA to separate the fragments.</p>
<p style="text-align: justify;">This means that nucleic acids have about the same charge-to-mass ratio, even if they differ in length.&nbsp;</p>
<p style="text-align: justify;">The smaller fragments of nucleotides move more easily, and hence faster, through the pores of the gel than the longer fragments do.</p>
<p style="text-align: justify;">Therefore, DNA fragments can be separated by size.&nbsp;</p>
<p style="text-align: justify;">The smaller the fragment, the farther it travels in a given period of time.</p>
</div>
</div>
<div class="card col-12">
<div class="card-body">
<p></p>
<p style="text-align: justify;">steps in gel electrophoresis</p>
<p style="text-align: justify;">Step 1. Prepare a gel (a thin slab of agarose), and place it in a gel box between two&nbsp;electrodes. The gel has wells for placing the DNA samples to be analyzed.&nbsp;Add a buffer to cover the gel.</p>
</div>
</div>
</div>
<p></p>
<hr>
<div class="d2l-element mceNonEditable" data-idx="4" data-type="left" role="section" aria-describedby="d2l-uid-2344">
<p id="d2l-uid-2344" class="sr-only">Timeline</p>
<ul class="timeline" data-prop="0|left">
<li>
<div class="timeline-badge"></div>
<div class="timeline-panel no-title">
<div class="timeline-heading">
<h3 data-itemprop="0|0"></h3>
</div>
<div class="timeline-body">
<div data-itemprop="0|1">
<p style="text-align: justify;">Step 2. Load DNA sample solutions, such as PCR products, into the wells of the&nbsp;gel, beside a well loaded with DNA “marker” fragments. A molecular marker&nbsp;is a DNA fragment of known size that is used to determine the lengths of&nbsp;unknown fragments by comparison. Multiple copies of many different-sized&nbsp;markers are run in a separate lane at the same time for comparison. The sizes&nbsp;of the fragments in the sample are determined by comparing their travelling&nbsp;distances to the travelling distances of the known molecular markers.<br>A special dye, called a loading dye, is added. The dye has two functions.&nbsp;First, because it is denser than the DNA, the dye keeps the DNA from&nbsp;floating out of the well. It sinks, along with the DNA, into the well. Second,&nbsp;the dye travels slightly faster than the fastest nucleotide fragments. The rate&nbsp;at which the dye is seen to move can be used to judge the pace of the electrophoresis process.</p>
</div>
</div>
</div>
</li>
<li>
<div class="timeline-badge"></div>
<div class="timeline-panel no-title">
<div class="timeline-heading">
<h3 data-itemprop="1|0"></h3>
</div>
<div class="timeline-body">
<div data-itemprop="1|1">
<p style="text-align: justify;">Step 3. Apply an electric current to the gel. DNA fragments are negatively charged,&nbsp;so they migrate toward the positive pole. Shorter DNA fragments migrate&nbsp;faster than longer DNA fragments. When the electric current is stopped,&nbsp;fragments of equal length will have travelled equal distances. The result will&nbsp;be clusters of equal-sized fragments forming invisible bands within the gel.<br>Step 4. To be able to see the DNA fragments once the process is complete, stain the&nbsp;electrophoresis gel. Ethidium bromide is one compound that can be used to&nbsp;stain the gel. It inserts itself inside the DNA double-helical ladder structure. When the gel is viewed under a UV light, the bands of DNA are visible because the ethidium bromide glows or&nbsp;fluoresces.</p>
</div>
</div>
</div>
</li>
<li>
<div class="timeline-badge"></div>
<div class="timeline-panel">
<div class="timeline-heading">
<h3 data-itemprop="2|0">Sequencing DNA</h3>
</div>
<div class="timeline-body">
<div data-itemprop="2|1">
<p style="text-align: justify;">The development of DNA technologies for analyzing genes has revolutionized&nbsp;experimental biology. DNA sequencing techniques (analyzing and determining the&nbsp;base sequences of DNA) have made it possible for researchers to analyze the base&nbsp;sequences of cloned genes and DNA fragments amplified by PCR. Having the complete sequence of a gene helps researchers understand how the gene functions. Using&nbsp;just the nucleotide sequence, they can identify mutations, locate regulatory sequences&nbsp;and gene sequences, and compare homologous genes of different species.<br>Many scientists have worked to develop methods for sequencing DNA&nbsp;(Section 8.5). One efficient method is called the chain termination method. It was&nbsp;developed by Frederick Sanger at Cambridge University in the 1970s. It relies on&nbsp;the addition of a labelled dideoxy nucleotide (ddNTP) to a growing DNA strand.<br>This nucleotide, when incorporated into the DNA, prevents the binding of the next&nbsp;nucleotide, thus terminating the elongation of the newly synthesized DNA strand.</p>
<p style="text-align: justify;">The labels on the dideoxy nucleotides are dyes that fluoresce and can be used to identify the specific base when exposed to laser light.</p>
</div>
</div>
</div>
</li>
<li>
<div class="timeline-badge"></div>
<div class="timeline-panel no-title">
<div class="timeline-heading">
<h3 data-itemprop="3|0"></h3>
</div>
<div class="timeline-body">
<div data-itemprop="3|1">
<p style="text-align: justify;">For the chain termination method in the laboratory, four reaction tubes&nbsp;are set up, each with many copies of normal deoxyribose nucleotides (dATP, dTTP, dGTP, and dCTP), multiple copies of the DNA to be sequenced, DNA polymerase, and a DNA primer. Only one of the labelled dideoxy nucleotides&nbsp;(ddATP, ddTTP, ddGTP, or ddCTP) is added to each tube. This dideoxy&nbsp;nucleotide is in a much smaller concentration than the normal nucleotide. One tube&nbsp;contains ddATP, another ddTTP, another ddGTP, and the final one ddCTP. Synthesis<br>of a complementary chain produces DNA fragments of different sizes, each with their&nbsp;last nucleotide labelled.&nbsp;</p>
<p style="text-align: justify;">The contents of the four reaction tubes are combined and then separated by&nbsp;gel electrophoresis with a resolution of only a single nucleotide. The sequence can&nbsp;then be determined by reading the order of the labelling type that corresponds&nbsp;to the specific dideoxy nucleotide. This process is now computer automated.&nbsp;</p>
</div>
</div>
</div>
</li>
<li>
<div class="timeline-badge"></div>
<div class="timeline-panel no-title">
<div class="timeline-heading">
<h3 data-itemprop="4|0"></h3>
</div>
<div class="timeline-body">
<div data-itemprop="4|1">
<p style="text-align: justify;">The entire human genome has now been sequenced, along with many other&nbsp;genomes. Many scientists worked on the project. They used two techniques, both&nbsp;involving the Sanger method for DNA sequencing. However, one technique, called&nbsp;the whole-genome shotgun method, proved to be faster than the other. This method was&nbsp;developed by Craig Venter of Celera Genomics. You will read more about Frederick&nbsp;Sanger and Craig Venter in Section 8.5.&nbsp;In the shotgun method, many copies of the DNA are randomly cut into tiny&nbsp;fragments by passing the DNA through a pressurized syringe. The fragments&nbsp;are cloned in plasmids and then sequenced using the Sanger method. Computer&nbsp;software is used to determine the overall sequence by analyzing the many partial sequences.&nbsp;</p>
<p style="text-align: justify;">The shotgun method is analogous to tearing&nbsp;10 copies of a book randomly into smaller sets of a few pages each and, by matching&nbsp;overlapping pages, reassembling a complete copy of the book with the pages in the<br>correct order.&nbsp;</p>
</div>
</div>
</div>
</li>
<li>
<div class="timeline-badge"></div>
<div class="timeline-panel">
<div class="timeline-heading">
<h3 data-itemprop="5|0">Analyzing Genomes</h3>
</div>
<div class="timeline-body">
<div data-itemprop="5|1">
<p style="text-align: justify;">Once the sequence of a whole genome has been derived, there are two approaches&nbsp;that can be used for its analysis: structural genomics (the study of the actual sequencing&nbsp;of genomes and the analysis of the nucleotide sequences to locate genes within the&nbsp;genome) and functional genomics (the study of the functions of genes, how they are&nbsp;expressed, what proteins they encode, and what role these proteins play in metabolic&nbsp;processes).</p>
<p style="text-align: justify;">Most research is focused on functional genomics because genes control the functions of cells, and therefore the functions of organisms. Functional genomics relieon bioinformatics: the combination of laboratory experiments and sophisticated&nbsp;computer analyses. Bioinformatics can be used, for example, to find a gene within a&nbsp;genomic sequence, align sequences in databases to determine the degree of matching,</p>
<p style="text-align: justify;">predict the structure and function of a gene product and postulate evolutionary relationships for sequences.</p>
<p style="text-align: justify;">Genes that code for proteins are particularly interesting in genome analysis. Once&nbsp;a genome is sequenced, researchers use computer algorithms to search both strands&nbsp;of the sequence for these genes. A genome can be translated by the computer to give&nbsp;the amino acid sequence of the protein it could encode. Researchers may then be able&nbsp;to assign a function to the amino acid sequence by performing a sequence similarity</p>
<p style="text-align: justify;">search: a computer-based comparison of a DNA or amino acid sequence with databases&nbsp;of sequences of known genes or proteins. If the sequence resembles that of a&nbsp;previously sequenced gene, the two genes are related in an evolutionary sense and are&nbsp;likely to have similar functions.</p>
</div>
</div>
</div>
</li>
<li>
<div class="timeline-badge"></div>
<div class="timeline-panel no-title">
<div class="timeline-heading">
<h3 data-itemprop="6|0"></h3>
</div>
<div class="timeline-body">
<div data-itemprop="6|1">
<p style="text-align: justify;">Analysis of genome sequencing has led to many new discoveries about genetic&nbsp;organization, and many earlier hypotheses have been verified or disproven. A surprising discovery is that the eukaryotic genomes sequenced to date contain large&nbsp;numbers of previously unknown genes—many more than scientists expected to&nbsp;find. In Caenorhabditis (a nematode), for example, 12 000 of the 19 000 genes have&nbsp;no known function. Identifying these genes and their functions is one of the major&nbsp;challenges for molecular geneticists. Another surprising discovery is the degree to&nbsp;which different organisms, some widely separated in their evolution, contain similar&nbsp;genes. For example, even though the yeast Saccharomyces, a single-celled fungus, is&nbsp;separated from our species by hundreds of millions of years of evolutionary history,&nbsp;about 2300 of its approximately 6000 genes are related to mammalian genes. Many&nbsp;of these related genes control progress through the cell cycle. The similarities are so&nbsp;close that the yeast and human versions of several genes can be interchanged with&nbsp;little or no effect on cell functions in either organism.</p>
<p style="text-align: justify;">Gene sequences also show that eukaryotic genomes contain large numbers of non-coding sequences, most of them in the form of repeated sequences of various lengths&nbsp;and numbers. Although these sequences make up about 25 to 50 % of the total genomic&nbsp;DNA in different eukaryotic species, their functions are unknown at this time.&nbsp;Nanopore Sequencing&nbsp;Nanopore sequencing is a revolutionary method that scientists have been working to&nbsp;perfect since 1995. It involves drawing individual strands of DNA through tiny sub-&nbsp;microscopic holes called nanopores and reading the DNA sequence, one base pair&nbsp;at a time, as the strand passes through the nanopore. All of the DNA bases (A, C, T,&nbsp;and G) fit precisely through a 2.5 nm hole. Some transmembrane cellular proteins&nbsp;work well for nanopore sequencing, although a tiny hole drilled in a super thin sheet&nbsp;of silicon has also been used. Each base is detected as it passes through the pore by&nbsp;an ingenious way of reading the conduction of a current across the silicon sheet. The&nbsp;current that passes is slightly different, depending on which base is passing through the&nbsp;nanopore at the time.&nbsp;</p>
</div>
</div>
</div>
</li>
<li>
<div class="timeline-badge"></div>
<div class="timeline-panel no-title">
<div class="timeline-heading">
<h3 data-itemprop="7|0"></h3>
</div>
<div class="timeline-body">
<div data-itemprop="7|1">
<p style="text-align: justify;">The development of nanopore sequencing is one of many efforts by genetic technologists around the world to win the “$1000 genome initiative.” The race is on to&nbsp;come up with a method that will allow any human to have his or her entire genome&nbsp;sequenced for a fee.</p>
<p style="text-align: justify;">Studying Gene Function Using DNA Microarrays&nbsp;A DNA microarray is a technology that helps scientists pinpoint the functions of specific genes, rather than just their locations, and to compare the expressions of genes in&nbsp;different types of cells or different organisms. Microarrays can be used to understand&nbsp;the processes involved in the development of an organism or to detect genetic mutations by comparing an individual’s DNA with so-called normal DNA.</p>
<p style="text-align: justify;">A microarray, or gene chip, is designed to hold many individual DNA samples,&nbsp;ranging from just 10 to 2.1 million. These DNA samples, called probes, are known&nbsp;sequences, and the location of each probe on the chip is also known. The technology&nbsp;that is used to create microarrays is similar to the technology that is used to lay out&nbsp;electronic circuits on a computer chip. The microarray protocol starts with the isolation of mRNA from the two types of cells to be compared: a reference cell (a normal cell) and an experimental cell (such as a cancer cell).</p>
</div>
</div>
</div>
</li>
<li>
<div class="timeline-badge"></div>
<div class="timeline-panel no-title">
<div class="timeline-heading">
<h3 data-itemprop="8|0"></h3>
</div>
<div class="timeline-body">
<div data-itemprop="8|1">
<p style="text-align: justify;">The extracted mRNA is used to build complementary DNA (cDNA) libraries. The&nbsp;cDNA from each type of cell is labelled with a specific dye, such as green dye for the&nbsp;reference cell and red dye for the experimental cell. The cDNA sequences are then&nbsp;denatured into single strands and placed onto the surface of the DNA microarray.&nbsp;The DNA microarray contains probes that correspond to the coding genes of the&nbsp;genome being studied. Any labelled cDNA that is complementary to the sequence of a&nbsp;probe will bind (hybridize) to this probe, while cDNA with no matching sequence will&nbsp;be washed off the chip. The fluorescence is then detected, and its location is recorded.&nbsp;Green spots mean that the reference cDNA was hybridized with the probe; hence, the&nbsp;gene for the corresponding protein is expressed in the normal cell. Red spots mean that&nbsp;the experimental cDNA was hybridized with the probe; in other words, the gene was&nbsp;active in the experimental cell (the cancer cell). Yellow spots mean that cDNA for both&nbsp;the reference cell and the experimental cell were hybridized. Both the normal cell and&nbsp;the cancer cell express the same gene. Researchers can then determine which genes&nbsp;have altered expressions in the cancer cell. The advantage of the microarray is that the&nbsp;expressions of thousands of genes can be identified simultaneously in a particular cell.</p>
</div>
</div>
</div>
</li>
<li>
<div class="timeline-badge"></div>
<div class="timeline-panel no-title">
<div class="timeline-heading">
<h3 data-itemprop="9|0"></h3>
</div>
<div class="timeline-body">
<div data-itemprop="9|1">
<p style="text-align: justify;">Genetically modified (GM) canola and other field crops have been engineered to be resistant to specific herbicides.&nbsp;</p>
<p style="text-align: justify;">This technology means that herbicides can be sprayed&nbsp;to kill weeds, but the crop plant remains healthy.&nbsp;</p>
<p style="text-align: justify;">The oil from the genetically modified canola plant is identical to conventional canola seed oil.&nbsp;</p>
<p style="text-align: justify;">Approximately 80 % of the western Canada canola crop is the genetically.&nbsp; modified variety.</p>
<p style="text-align: justify;">Farmers spend about 40 % less on herbicides for GM canola crops, versus conventional canola crops, and they report up to a 10 % increase in yield.</p>
</div>
</div>
</div>
</li>
</ul>
</div>
<p>&nbsp;</p>
<hr>
<div class="d2l-element mceNonEditable" data-idx="3" data-type="noicon" role="section" aria-describedby="d2l-uid-2510">
<p id="d2l-uid-2510" class="sr-only">Callout</p>
<div class="card card-standard" data-prop="0|noicon">
<div class="card-body">
<div class="card-text">
<h4 data-prop="1|null"><!-- Empty --></h4>
<div data-prop="2|null">
<p style="text-align: justify;">An advantage of GM plants is their potential to provide food for millions of hungry people in the developing world.&nbsp;</p>
<p style="text-align: justify;">&nbsp;A major drawback of GMOs is the increased cost of the patented seeds, making them unaffordable to some farmers.&nbsp;</p>
<p style="text-align: justify;">&nbsp;For a family in the developing world, however, having higher-yielding and pest-resistant crops might mean the difference between hunger and a better life.</p>
</div>
</div>
</div>
</div>
</div>
</div>
<div class="col-12"><footer><!-- <p>© [Client] [Year]</p> -->
<p><img src="./EVA%20SCHOOL1.png" alt="" title="" data-d2l-editor-default-img-style="true" style="max-width: 100%;"></p>
</footer></div>
</div>
</div>
</div>
<p>
<script src="./jquery-3.5.1.min.js"></script>
<script src="./popper.min.js"></script>
<script src="./bootstrap.min.js"></script>
</p>
<!-- Template JavaScript -->
<p>
<script src="./scripts.min.js"></script>
</p></body></html>